easybuild/easyconfigs/o/OpenFOAM/OpenFOAM-v1706-foss-2016b.eb

```diff
@@ -0,0 +1,44 @@
+# -*- python -*-
+name = 'OpenFOAM'
+version = 'v1706'
+
+homepage = 'http://www.openfoam.com/'
+description = """OpenFOAM is a GPL-opensource C++ CFD-toolbox.
+This offering is not approved or endorsed by OpenCFD Ltd,
+producer and distributor of the OpenFOAM software via www.openfoam.com,
+and owner of the OPENFOAM trademark.
+"""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'cstd': 'c++11'}
+
+source_urls = [
+    ('https://sourceforge.net/projects/openfoamplus/files/%(version)s',
+     'download')
+]
```

Comment:
We have automated style checks in place too now, but allow lines up to 120 chars; too much silly splits (like this), we're not in the 80s anymore so 120 chars makes more sense imho ;)